pkg/storaged/format-dialog.jsx

```diff
@@ -192,20 +198,34 @@
 
     const crypto_split_options = parse_options(crypto_options);
     extract_option(crypto_split_options, "noauto");
+    extract_option(crypto_split_options, "nofail");
+    extract_option(crypto_split_options, "_netdev");
     const crypto_extra_options = unparse_options(crypto_split_options);
 
     let [, old_dir, old_opts] = get_fstab_config(block, true);
-    if (!old_opts || old_opts == "defaults")
+    if (!old_opts)
         old_opts = initial_mount_options(client, block);
 
     const split_options = parse_options(old_opts == "defaults" ? "" : old_opts);
     extract_option(split_options, "noauto");
     const opt_ro = extract_option(split_options, "ro");
     const opt_never_auto = extract_option(split_options, "x-cockpit-never-auto");
+    const opt_nofail = extract_option(split_options, "nofail");
+    const opt_netdev = extract_option(split_options, "_netdev");
     const extra_options = unparse_options(split_options);
 
     let existing_passphrase_type = null;
 
+    let at_boot;
+    if (opt_never_auto)
+        at_boot = "never";
+    else if (opt_netdev)
+        at_boot = "netdev";
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221125-094531-d55c9eb8-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)